### DISCUSSION TITLE
Right click > remove playlist warns before removing, while middle click on a playlist tab does not warn, is this behavior consistent?

### DISCUSSION THREAD

**Discussion by Retia-Adolf** [DeaDBeeF-Player/deadbeef#2897] [General]
I think this might likely lead user to remove playlist by mistake, thinking that scrolling between multiple playlist tabs also uses the middle wheel.

**Comment by Oleksiy-Yakovenko** [comment]
Seems like a bug, thanks -- will add for tracking
